features/achordion.c

```diff
@@ -219,7 +234,9 @@ bool process_achordion(uint16_t keycode, keyrecord_t* record) {
 
 #ifdef ACHORDION_STREAK
   // update idle timer on regular keys event
-  streak_timer = (timer_read() + achordion_streak_timeout(keycode)) | 1;
+  if (achordion_state != STATE_HOLDING) {
+    streak_timer = record->event.time | 1;
```

Comment:
Is this line needed? If I haven't gotten confused, it is only reached when there's a non-tap-hold key event while a tap-hold key is settled, but not yet released. Maybe removing this line would help fast hotkeys like the "Ctrl-V, Ctrl-V" edge case you mentioned.

Reply:
This is exactly to _help_ the ctrl-v ctrl-v situation. I don't want any keys pressed during a mod-combo to update the streak timer, since they aren't regular typing taps.

Reply:
Aha, right, I got that backwards. Thanks for the correction!

Reply:
I realized this was applying when shift was held, and I wanted capital letters to count as normal streak typing, so I ended up adding the function which lets you customize which keys count towards a streak.